packages/client/src/getStorageNodeRegistryFromContract.ts

```diff
@@ -1,40 +0,0 @@
-import { JsonRpcProvider } from '@ethersproject/providers'
-import { Contract } from '@ethersproject/contracts'
-import { createStorageNodeRegistry, StorageNodeInfo, StorageNodeRegistry } from 'streamr-client-protocol'
```

Comment:
Now that this file is deleted, I wonder if we are using `StorageNodeRegistry` from protocol anywhere (else than tests)? Maybe it can be removed as well...

Reply:
never mind :D